src/components/file-dropzone/file-dropzone.tsx

```diff
@@ -0,0 +1,123 @@
+import {
+    Component,
+    h,
+    Event,
+    EventEmitter,
+    Host,
+    Element,
+    Prop,
+} from '@stencil/core';
+import { FileInfo } from 'src/interface';
+import { createRandomString } from 'src/util/random-string';
```

Comment:
are we gonna use this `createRandomString`, or use the `crypto.randomUUID()` instead?

https://github.com/Lundalogik/lime-elements/blob/9ec29350164b3d7dfa1c03de6bae386a9d8d6c7e/src/components/chip/chip.tsx#L173

Reply:
The createRandomString function uses crypto.randomUUID() but also has a fallback in case the browser being used doesn't support crypto or have it available.

Reply:
I was about to say that we don't need to support browsers that don't support `crypto.randomUUID()`, but thought I'd look at caniuse.com first, just to be sure, and it looks like `crypto.randomUUID()` only got support in Chrome and Firefox on Android during February _this year_ 😲

<img width="1440" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/154725/90f15590-5f1d-42cf-a866-5157846b34f9">

https://caniuse.com/?search=crypto.randomUUID

Although, I wonder if the lack of red blobs for earlier versions doesn't just mean they don't have data for those versions (though that would also surprise me… so I don't really know what to think… 🤷‍♂️)

We were using `createRandomString()` in an example, and I [replaced](https://github.com/Lundalogik/lime-elements/commit/b88977369b91696d1c1649293a03c35178fe85de) that with `crypto.randomUUID()` just this last Friday, because we don't export `createRandomString`, so it shouldn't be used in examples. (In this specific example, the value was used for the `key` attribute, so I didn't have to prepend the UUID with a letter, but when used with the `id` attribute, remember that IDs must begin with a letter, not a number, so the UUID would need to be prepended, like it is in `createRandomString`.)

I'll see if BrowserStack has older Chrome versions on Android that I can use to check…

Reply:
I've added `createRandomString() `as an exported function to a` utils.ts` in` lime-crm-components` in the PR I'm working on related to the new FileUpload class but maybe it should instead be in `lime-web-components` if we'll possibly need it elsewhere?

Reply:
BrowserStack are running Chrome 120 on all the Android devices I tested just now, and `crypto.randomUUID()` works on all of them. I think we should just stop using `createRandomString`, rather than spread it to more places.

Reply:
I'll remove it from my PR in that case. I think actually now that I'm cleaning it up, I'm no longer using it anyways as we went a different direction for handling uploads.

Reply:
I agree with you all, please see https://github.com/Lundalogik/lime-elements/pull/2802/commits/aba85d16f2e3b4bd43bdc25e988c790914711e1a